back/src/main/java/com/ubikloadpack/jmeter/ulp/observability/listener/ULPObservabilityListener.java

```diff
@@ -224,7 +224,7 @@ public void setRegex(String regex) {
 	}
 
 	public String getTotalLabel() {
-		return getPropertyAsString(ULPODefaultConfig.TOTAL_LABEL_PROP, ULPODefaultConfig.totalLabel());
+		return Util.makeOpenMetricsName(getPropertyAsString(ULPODefaultConfig.TOTAL_LABEL_PROP, ULPODefaultConfig.totalLabel()));
```

Comment:
Is this related to the PR ? or should you rebase  your branch ?

Reply:
The rebase is done. I took some times to understand what it do because i never used it. Now the PR should only has the new commits.